homeassistant/components/steam_online/sensor.py

```diff
@@ -140,10 +147,28 @@ def _get_current_game(self):
         _LOGGER.error("Unable to find name of app with ID=%s", game_id)
         return repr(game_id)
 
+    def _get_last_online(self):
+        last_online = datetime.fromtimestamp(
+            mktime(self._profile.last_online)
+        ).isoformat()
+
+        if last_online:
+            return last_online
+
+    def _get_level(self):
```

Comment:
This doesn't add an benefit. 

In line 114 do something like:

```python
self._level = self._profile.level

Reply:
THanks for the suggestion, removed the function and changed to direct assignation as suggested